.github/workflows/labeler.yml

```diff
@@ -1,18 +0,0 @@
-name: "Pull Request Labeler"
-on:
-  - pull_request_target
-concurrency:
-  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
-  cancel-in-progress: true
```

Comment:
We have removed this. It would probably cause some waste runs. Not sure how significant that would be